slsim/Deflectors/cluster_catalog_lens.py

```diff
@@ -0,0 +1,281 @@
+import numpy as np
+import numpy.random as random
+from slsim.selection import object_cut
+from slsim.Deflectors.richness2mass import mass_richness_relation
+from slsim.Deflectors.halo_population import gene_e_ang_halo, concent_m_w_scatter
+from colossus.cosmology import cosmology as colossus_cosmo
+from slsim.Deflectors.velocity_dispersion import vel_disp_abundance_matching
+from slsim.Deflectors.elliptical_lens_galaxies import (
+    elliptical_projected_eccentricity,
+    vel_disp_from_m_star,
+)
+from slsim.Deflectors.velocity_dispersion import vel_disp_nfw
+from slsim.Deflectors.deflectors_base import DeflectorsBase
+from lenstronomy.Util.param_util import phi_q2_ellipticity
+from astropy import units as u
+from astropy.table import hstack
+from scipy.spatial.distance import cdist
+
+
+class ClusterCatalogLens(DeflectorsBase):
```

Comment:
Do you want to change this name to ClusterDeflectors? If you prefer ClusterCatalogLens, I am ok with it.

Reply:
Yes, ClusterDeflectors is much better